suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -96,7 +97,7 @@ export const XpubScanScreen = ({
                         setIsHintSheetVisible(true);
                     },
                 });
-            }, 1000);
+            }, 2000);
```

Comment:
nit: Is this timeout increased due to performance? Would be nice to expand on the comment so it's clear what changing the value could possibly affect

Reply:
I will revert that, leftovers from debug